src/Samples/UiBoundCmdParam/Program.fs

```diff
@@ -26,7 +26,7 @@ let bindings () : Binding<Model, Msg> list = [
   "Numbers" |> Binding.oneWay(fun m -> m.Numbers)
   "Limit" |> Binding.twoWay((fun m -> float m.EnabledMaxLimit), int >> SetLimit)
   "Command" |> Binding.cmdParamIf(
-    (fun p m -> Command),
+    (fun _ _ -> Command),
```

Comment:
I'd like this reverted.

My reasoning is that the names of even unused parameters in bindings can make signatures/intent more clear to newcomers (who are the primary target audience for the samples).

Reply:
Sure.  Done.